exercises/Expressions-variables-4-R.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: exercise
+title: Expressions and Variables 5
+language: R
+---
+
+The following code calculates the total net primary productivity (NPP)
+per day for two sites based on the grams of carbon produced per square
+meter per day, and the total area of the sites, and prints them out.
+
+    site1_g_carbon_m2_day = 5
+    site2_g_carbon_m2_day = 2.3
+    site1_area_m2 = 200
+    site2_area_m2 = 450
+    site1_npp_day = site1_g_carbon_m2_day * site1_area_m2 
+    site2_npp_day = site2_g_carbon_m2_day * site2_area_m2
+    print(site1_npp_day)
+    print(site2_npp_day)
```

Comment:
Done.

On Wed, Apr 15, 2015 at 4:28 PM, Ethan White <notifications@github.com>
wrote:

> In exercises/Expressions-variables-4-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/13#discussion_r28464852>
> :
>
> > +title: Expressions and Variables 5
> > +language: R
> > +---
> > +
> > +The following code calculates the total net primary productivity (NPP)
> > +per day for two sites based on the grams of carbon produced per square
> > +meter per day, and the total area of the sites, and prints them out.
> > +
> > +    site1_g_carbon_m2_day = 5
> > +    site2_g_carbon_m2_day = 2.3
> > +    site1_area_m2 = 200
> > +    site2_area_m2 = 450
> > +    site1_npp_day = site1_g_carbon_m2_day * site1_area_m2
> > +    site2_npp_day = site2_g_carbon_m2_day * site2_area_m2
> > +    print(site1_npp_day)
> > +    print(site2_npp_day)
>
> This code chunk should be in back ticks rather than indented.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/13/files#r28464852>
> .
>